.github/workflows/previews.yml

```diff
@@ -10,6 +10,10 @@ jobs:
   build:
     # Operating system to run job on
     runs-on: ubuntu-latest
+    # Stops previous workflow runs in the event of multiple pushes to a PR
+    concurrency:
+      group: previews-group
```

Comment:
This is super cool. :) 

I think by hard coding a group that means that this workflow will only run the most recent workflow start.